assets/javascripts/modules/checkout/planDateFilter.jsx

```diff
@@ -16,13 +14,13 @@ define([
                 sixday: (date) => date.day() === 1,
                 sunday: (date) => date.day() === 6,
                 weekend: (date) => date.day() === 6,
-                default: (date) => date.day() === 1
+                allDays: (date) => date.day() === 1
```

Comment:
I just thought 'allDays' made more sense given the other enums, also default is a keyword and lit up my IntelliJ linter.